src/agent/Makefile

```diff
@@ -23,14 +23,38 @@ COMMIT_MSG = $(if $(COMMIT),$(COMMIT),unknown)
 # Exported to allow cargo to see it
 export VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
+include ../../utils.mk
+
 EXTRA_RUSTFEATURES :=
 
 ##VAR SECCOMP=yes|no define if agent enables seccomp feature
 SECCOMP := yes
 
+##VAR WASM=yes|no define if agent enables wasm feature
+WASM := yes
+## wasmer is just support x86, x86_64, aarch64, arm, riscv64, riscv32
+ifneq ($(ARCH),x86_64)
+ifneq ($(ARCH),x86)
+ifneq ($(ARCH),aarch64)
+ifneq ($(ARCH),arm)
+ifneq ($(ARCH),riscv64)
+ifneq ($(ARCH),riscv32)
+	WASM = no
+endif
+endif
+endif
+endif
+endif
+endif
+
 # Enable seccomp feature of rust build
 ifeq ($(SECCOMP),yes)
-    override EXTRA_RUSTFEATURES += seccomp
+    override EXTRA_RUSTFEATURES := $(EXTRA_RUSTFEATURES)seccomp,
```

Comment:
Is a space needed between `$(EXTRA_RUSTFEATURES)` and `seccomp`?

Reply:
Updated.